saleor/plugins/avatax/__init__.py

```diff
@@ -360,24 +360,29 @@ def get_order_lines_data(
             amount=undiscounted_amount,
         )
 
-        if undiscounted_amount != price_with_discounts_amount:
+        # for invoice transaction we want to include only undiscounted price
```

Comment:
Are you sure of this description? I thought that for invoice transactions we want to only include a "final" price, like price_with_discounts